static/app/views/projects/projectContext.tsx

```diff
@@ -140,7 +140,7 @@ class ProjectContext extends Component<Props, State> {
   );
 
   unsubscribeMembers = MemberListStore.listen(
-    (memberList: MemberListStore['state']) => this.setState({memberList}),
+    (memberList: typeof MemberListStore['state']) => this.setState({memberList}),
```

Comment:
Does this even need typed?

Reply:
Not sure, but looks like other listeners type it similarly.

Reply:
nvm yeah I doubt it does